k8s/templates/frontend-pgbouncer.yaml

```diff
@@ -10,12 +10,16 @@ data:
   # Aron: I configured pgbouncer based on the settings
   # recommended by this article:
   # https://medium.com/futuretech-industries/postgres-raffle-10k-connections-35-mo-part-two-b4c2e0c86e37
-  POOL_MODE: "transaction"
+  # https://docs.djangoproject.com/en/1.11/ref/databases/#transaction-pooling-and-server-side-cursors
+  POOL_MODE: "session"
   SERVER_RESET_QUERY: "DISCARD ALL"
```

Comment:
This `SERVER_RESET_QUERY` isn't actually used in `transaction` pooling mode, without `SERVER_RESET_QUERY_ALWAYS = 1`. https://www.pgbouncer.org/config.html#server_reset_query_always